src/main/java/org/molgenis/downloader/api/metadata/MolgenisVersion.java

```diff
@@ -54,18 +55,6 @@ public boolean equalsOrSmallerThan(final MolgenisVersion that) {
     return this.equals(that) || this.smallerThan(that);
   }
 
-  public int getMajor() {
-    return major;
-  }
-
```

Comment:
why is this removed?

Reply:
Unused code

Reply:
are the private fields still relevant then?

Reply:
Yes, they're used in the other methods